content/src/main/content/jcr_root/_oak_index/.content.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:oak="http://jackrabbit.apache.org/oak/ns/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0" jcr:primaryType="nt:unstructured">
+        <cqAuditEvent
```

Comment:
@davidjgonzalez -- Sounds like a plan. I'm thinking that it might make sense to add a warning or something as well if no index is found because it's pretty worthless without the index. I'll give it a crack and update here when complete